src/ui/components/Badge/index.js

```diff
@@ -0,0 +1,17 @@
+/* @flow */
+import React from 'react';
+
+import './styles.scss';
+
+type Props = {|
+  label: string,
+  type?: string,
```

Comment:
This should be a [literal type](https://flow.org/en/docs/types/literals/) that only accepts a single string, e.g. `type?: 'featured'`. As we add more types we can adjust it. Also, sadly, I think you should also throw an error if an unexpected type is passed in. This is because we don't have great Flow coverage yet.